crates/gpui/src/platform/windows/direct_write.rs

```diff
@@ -0,0 +1,1286 @@
+use std::{borrow::Cow, sync::Arc};
+
+use ::util::ResultExt;
+use anyhow::{anyhow, Result};
+use collections::HashMap;
+use itertools::Itertools;
+use parking_lot::{RwLock, RwLockUpgradableReadGuard};
+use smallvec::SmallVec;
+use windows::{
+    core::{implement, HSTRING, PCWSTR},
+    Foundation::Numerics::Matrix3x2,
+    Win32::{
```

Comment:
Can we change these windows crate imports to use the `*` syntax we use in other crates?

Reply:
Done.